hail/python/hailtop/batch/job.py

```diff
@@ -1107,15 +1070,31 @@ def wrapped(*args, **kwargs):
 import traceback
 import json
 import sys
-
-with open(\\"{result}\\", \\"wb\\") as dill_out:
+from time import perf_counter
```

Comment:
Guessing this was for testing. Can remove this right?

Reply:
Removed!